pypi_formula_mappings.json

```diff
@@ -36,6 +36,9 @@
   "certbot": {
     "extra_packages": ["certbot-apache", "certbot-nginx"]
   },
+  "cfn-lint": {
+    "exclude_packages": ["junit-xml"]
```

Comment:
adding this python formula mapping is helpful for doing `brew update-python-resources cfn-lint`, while we still need `junit-xml` for installation.